src/client/features/search/index.js

```diff
@@ -192,26 +192,42 @@ class Search extends React.Component {
     const landing = (state.landingLoading ) ?
       h('div.search-landing.innner',[h(Loader, { loaded:!state.landingLoading , options: { color: '#16A085',position:'relative', top: '15px' }})]):
       state.landing.map(box=>{
+
         let synonyms=null;
         if(_.hasIn(box,'protein.alternativeName')){ 
-          const synonymsText= box.protein.alternativeName.map(obj => obj.fullName.value).join(', ');
-          synonyms=h('i.search-landing-small',synonymsText.slice(0,synonymsText.indexOf(',',70)));
+          const synonymsLength=115;
```

Comment:
I think all of this logic about hiding/showing more information can be handled by css.  We can go over this in person.